test/integration/features/F004_Error_Handling.feature

```diff
@@ -72,7 +72,6 @@ Feature: F004. Error Handling
     Examples: Organisation types
       | org_type |
       | dentist  |
-      | pharmacy |
```

Comment:
We will need the 'pharmacy' data for this scenario. Not sure why its been removed

Reply:
The scenario doesn't detect what's enabled or disabled. I couldn't figure out why this test was present without a trigger to verify that they should/shouldn't be enabled otherwise it's going to cause a failure unless both pharmacy and dentist are turned off. I can add pharmacy back, but it will fail as a result.